internal/clients/idler.go

```diff
@@ -32,9 +32,26 @@ type clusterView struct {
 	AppDNS string
 }
 
+type responseError struct {
+	Code        errorCode `json:"code"`
+	Description string    `json:"description"`
+}
+
+type jenkinsInfo struct {
+	State string `json:"state"`
```

Comment:
the State needs to be a type like the one here so that it is type-safe.

https://github.com/sthaha/fabric8-jenkins-proxy/commit/97d7357d13e5543d9878d6018c4e09c9a90ee060